hexa/workspaces/tests/test_schema.py

```diff
@@ -466,7 +484,253 @@ def test_get_workspace_members(self):
+                    workspaceMembership {
+                        id
+                        role
+                    }
+                }
+            }
+
+            """,
+            {
+                "input": {
+                    "workspaceId": str(self.WORKSPACE.id),
+                    "membershipId": str(uuid.uuid4()),
+                    "role": WorkspaceMembershipRole.EDITOR,
+                }
+            },
+        )
+        self.assertEqual(
+            {
+                "success": False,
+                "errors": ["MEMBERSHIP_NOT_FOUND"],
+                "workspaceMembership": None,
+            },
+            r["data"]["updateWorkspaceMemberRole"],
+        )
+
+    def test_update_workspace_member_role_workspace_not_found(self):
+        self.client.force_login(self.USER_ADMIN)
+        r = self.run_query(
+            """
+            mutation updateWorkspaceMemberRole($input: UpdateWorkspaceMemberRoleInput!) {
+                updateWorkspaceMemberRole(input: $input) {
+                    success
+                    errors
+                    workspaceMembership {
+                        id
+                        role
+                    }
+                }
+            }
+
+            """,
+            {
+                "input": {
+                    "workspaceId": str(uuid.uuid4()),
+                    "membershipId": str(self.WORKSPACE_MEMBERSHIP.id),
+                    "role": WorkspaceMembershipRole.EDITOR,
+                }
+            },
+        )
+        self.assertEqual(
+            {
+                "success": False,
+                "errors": ["WORKSPACE_NOT_FOUND"],
+                "workspaceMembership": None,
+            },
+            r["data"]["updateWorkspaceMemberRole"],
+        )
+
+    def test_update_workspace_member_role_permission_denied(self):
+        self.client.force_login(self.USER_REBECCA)
+        r = self.run_query(
+            """
+            mutation updateWorkspaceMemberRole($input: UpdateWorkspaceMemberRoleInput!) {
+                updateWorkspaceMemberRole(input: $input) {
+                    success
+                    errors
+                    workspaceMembership {
+                        id
+                        role
+                    }
+                }
+            }
+
+            """,
+            {
+                "input": {
+                    "workspaceId": str(self.WORKSPACE.id),
+                    "membershipId": str(self.WORKSPACE_MEMBERSHIP.id),
+                    "role": WorkspaceMembershipRole.EDITOR,
+                }
+            },
+        )
+        self.assertEqual(
+            {
+                "success": False,
+                "errors": ["PERMISSION_DENIED"],
+                "workspaceMembership": None,
+            },
+            r["data"]["updateWorkspaceMemberRole"],
+        )
+
+    def test_update_workspace_member_role(self):
+        self.client.force_login(self.USER_ADMIN)
```

Comment:
Could you use a normal user to do the tests ? It's not recommended to use a superuser when not required as it could have by-passes for almost everything.